lib/Caxy/HtmlDiff/ListDiff.php

```diff
@@ -0,0 +1,366 @@
+<?php
+
+namespace Caxy\HtmlDiff;
+
+class ListDiff extends HtmlDiff
+{
+    protected $listWords = array();
+    protected $listTags = array();
+    protected $listIsolatedDiffTags = array();
+    protected $isolatedDiffTags = array (
+        'ol' => '[[REPLACE_ORDERED_LIST]]',
+        'ul' => '[[REPLACE_UNORDERED_LIST]]',
+        'dl' => '[[REPLACE_DEFINITION_LIST]]',
+    );
+    
+    protected $listType; // holds the type of list this is ol, ul, dl
```

Comment:
For these comments, we can use docblocks instead of inline comments since these are properties on the class. We can also include `@var` annotations for each of these to help with auto-completion & analysis.